ui-tests/pom.xml

```diff
@@ -29,6 +29,13 @@
             <artifactId>uitestframework</artifactId>
             <version>${uitestframeworkVersion}</version>
         </dependency>
+        
```

Comment:
Is this dependency really needed here? I think the selenium dependency inside uitestframework pom is the one to update.